src/components/AppContext.js

```diff
@@ -51,20 +61,70 @@ class Provider extends Component {
   }
 
   clearFilters = () => {
-    this.setState(initialState)
+    const { state } = this
+    state.filterOpen = null
+    state.filters = {
```

Comment:
Mutation of `state` like this is not so kosher in react apps. We could make it safe by using `immer` but I think in this case sticking to the more idiomatic functional style would be fine.

i.e.

```js
this.setState({
  ...this.state,
  filterOpen: null,
  filters: initialState.filters,
})
```

Reply:
Spent a few hours on saturday figuring out this problem, but when you update state it also mutates the initial state object, so you cannot use it to reset state